doc/governance/org-docs/CHARTER.md

```diff
@@ -0,0 +1,70 @@
+# Charter for the HoloViz Organization
+
+This is the organizational charter for the HoloViz Organization. The HoloViz Organization is an open-source and collaborative effort with a mission to develop solutions for a wide range of visualization problems in Python.
+
+## 1. Terminology
+- "HoloViz Organization" or "Organization": The entire HoloViz ecosystem of Contributors and Projects
+- "HoloViz GitHub Organization": The Github organization at https://github.com/holoviz
+- "HoloViz" repo: The specific repository at https://github.com/holoviz/holoviz
+- "HoloViz.org": The website at https://holoviz.org/
+
+## 2. Organization Steering Committee
+By adding their name to the [Steering Committee.md file](./STEERING-COMMITTEE.md), Steering Committee members agree to the policies in this document.
+
+**2.1 Purpose**. The Steering Committee will be responsible for all technical oversight, project approval and oversight, policy oversight, and trademark management for the Organization.
+
+**2.2 Composition**. The Steering Committee voting members are listed in the steering-committee.md file in the repository. Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee. The Steering Committee will appoint a Chair responsible for organizing Steering Committee activity. There is no upper limit on the number of Steering Committee members.
+
+## 3. Voting
+
+**3.1. Decision Making**. The Steering Committee will strive for all decisions to be made by consensus. While explicit agreement of the entire Steering Committee is preferred, it is not required for consensus. Rather, the Steering Committee will determine consensus based on their good faith consideration of a number of factors, including the dominant view of the Steering Committee and nature of support and objections. The Steering Committee will document evidence of consensus in accordance with these requirements. If consensus cannot be reached, the Steering Committee will make the decision by a vote.
+
+**3.2. Voting**. The Steering Committee Chair will call a vote with reasonable notice to the Steering Committee, setting out a discussion period and a separate voting period. Any discussion may be conducted in person or electronically by text, voice, or video. The discussion will be open to the public. In any vote, each voting representative will have one vote. Except as specifically noted elsewhere in this Charter, decisions by vote require a simple majority vote of all reachable and vote-participating members.
```

Comment:
Should this describe policies around getting quorum?

Reply:
It certainly can! I haven't included that yet. What about adopting something like the [conda-forge quorum policies](https://conda-forge.org/docs/orga/governance.html#:~:text=the%20core%20group-,Quorum,-%3A)?

Reply:
The conda-forge rules sound really complicated, which I guess is appropriate for their purposes of adjucating between stakeholders with widely divergent interests. So far, disputes in HoloViz haven't seemed to need a solution that complex. Can we just define quorum as half of the active members of the relevant group, so that by definition inactive members don't make it impossible to reach quorum?  We could further put a minimum number of people (2?) required for any decision to be binding.

Reply:
That sounds good to me. I just added the suggested quorum rules to the org and project levels:

> Unless otherwise stated, a quorum for a vote can be met with at least 50% of active members, and at minimum, 2 participants. For example, in a vote requiring a simple majority, with 10 active members, at least 5 must vote; if 5 vote, there must be 3 affirmative votes. If there are 3 active members, at least 2 must vote; if 2 vote there must be 2 affirmative votes.

Reply:
Will the vote be done on Github?